packages/cadl-ranch-specs/http/models/inheritance/main.cadl

```diff
@@ -1,60 +1,112 @@
-import "@cadl-lang/rest";
 import "@azure-tools/cadl-azure-core";
 import "@azure-tools/cadl-ranch-expect";
 
 using Cadl.Rest;
 using Cadl.Http;
 using Azure.Core.Foundations;
 
-@doc("Illustrates round-trip polymorphic models.")
-@scenarioService("/polymorphic")
-namespace BasicPolymorphicModels;
+@serviceTitle("ComplexInheritanceClient")
+@serviceVersion("1.0.0")
+@server("http://localhost:3000", "Testserver endpoint")
+@doc("Illustrates inheritance model.")
+@route("/inheritance")
+namespace Model.Inheritance;
 
-@doc("Example base type.")
-model BaseClass {
-  @doc("An example property.")
-  baseClassProperty: string;
+@doc("Base model for non-discriminator three levels inheritance.")
+model Pet {
+  name?: string;
 }
 
-model DerivedFromBaseClassA extends BaseClass {
-  @doc("An example property on a derived type")
-  derivedClassAProperty: string;
+@doc("The second level model A in the non-discriminator three levels inheritance.")
```

Comment:
Update doc, and I think `age` can just be required. To be honest most of these properties can be required unless you're specifically trying to test optionality